app/views/courses/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= render 'pickCourse'%>
+
+<div class="modal fade" id="createCourse" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="exampleModalLabel">Create a new course</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+                
+        <%= form_for @course, url: {action: "create"} do |f| %>
```

Comment:
For better aesthetic can consider moving this to before `<div class="modal-body">`